### ISSUE TITLE
rewind differs between trajectory readers

### ISSUE THREAD

**Issue by fiona-naughton** [MDAnalysis/mdanalysis#902]
The `rewind` method in the auxiliary readers (which returns to the first frame of the trajectory) is implemented differently between readers.

In the base `ProtoReader` it looks like:
```Python
def rewind(self):
    """Position at beginning of trajectory"""
    self._reopen()
    self.next()
```
(`reopen()` is defined for each reader individually, to position before the first frame; and `next` is defined in the `ProtoReader`).

Several of the readers (GMS, TRJ, XYZ) overwrite this with exactly the same function. The PDB and XDR (and so XTC and TRR) readers overwrite it to instead call just `_read_frame(0)`; `_read_frame()` is defined in each reader individually, and doesn't call `next`.

Basically, this means that changes to ProtoReader's `next` to change/add to what happens when reading a new frame of any trajectory (e.g. updating auxiliaries in #868) won't apply when rewinding a pdb/trr/xtc trajectory; and any changes to `rewind` in the `ProtoReader` would not propagate through all readers. 

In #868, I've currently edited the PDB/XDR readers individually to fix the auxiliaries-not-updating case, but this is obviously not ideal. 

(Swapping `_read_next_frame()` for `next()` in the XDRBaseReader's `_read_frame()` should fix that problem there; but beyond that, I'm not sure on the best way to avoid this.)

**Comment by jdetle** [comment]
Hi, I think 871 applies to this. I'm not sure if I'll be able to get to it this week so if you want to work on it feel free.
